### ISSUE TITLE
code-block documentation doesn't mention `:class:` option

### ISSUE THREAD

**Issue by simonw** [sphinx-doc/sphinx#10461]
### Describe the bug

The following can be used to add an extra HTML class to a code block:
```
.. code-block:: json
   :class: metadata-json

    {
        "databases": {}
    }
```
But it's not mentioned in the documentation here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block

### How to Reproduce

View https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block

### Your project

https://github.com/simonw/datasette

### OS

All

### Python version

All

### Sphinx version

All

**Comment by simonw** [comment]
Looks like this is the file that should include this: https://github.com/sphinx-doc/sphinx/blob/master/doc/usage/restructuredtext/directives.rst#showing-code-examples

**Comment by tk0miya** [comment]
Thank you for your report. Fixed by #10464. Will be updated soon.
